addons/mail/wizard/mail_wizard_invite_views.xml

```diff
@@ -13,7 +13,8 @@
                         <field name="res_id" invisible="1"/>
                         <field name="partner_ids" widget="many2many_tags_email"
                                 placeholder="Add contacts to notify..."
-                                context="{'force_email':True, 'show_email':True}"/>
+                                options="{'no_quick_create': True}"
+                                context="{'show_email':True, 'form_view_ref': 'base.view_partner_simple_form'}"/>
```

Comment:
Cosmetic changes (since you alter the line) 😇 

```suggestion
                                context="{'show_email': True, 'form_view_ref': 'base.view_partner_simple_form'}"/>
```